systems/nerf.py

```diff
@@ -31,6 +32,7 @@ def forward(self, batch):
         return self.model(batch['rays'])
     
     def preprocess_data(self, batch, stage):
+        self.rank = get_rank()
```

Comment:
`self.rank` has already been set in `BaseSystem` so this is not needed:
https://github.com/bennyguo/instant-nsr-pl/blob/1f2ef5b0debfc095304afb8a973eb0aa884d3c02/systems/base.py#L18